sormas-rest/src/main/java/de/symeda/sormas/rest/AggregateReportResource.java

```diff
@@ -45,6 +47,13 @@
 		return result;
 	}
 
+	@POST
+	@Path("/indexList")
+	public List<AggregatedCaseCountDto> getIndexList(AggregateReportCriteria criteria) {
+		List<AggregatedCaseCountDto> result = FacadeProvider.getAggregateReportFacade().getIndexList(criteria);
```

Comment:
Nit pick, but return the result directly :) One warning less in the code :smiley: